Clients/GUI/Components/Wizard.razor

```diff
@@ -0,0 +1,26 @@
+﻿<CascadingValue Value="this">
+    <div id="@Id">
+        <ul class="nav nav-pills nav-justified">
+            @foreach (var step in Steps)
+            {
+                <li id="step-@(StepsIndex(step) + 1)" class="nav-item">
+                    <a class="nav-link @((ActiveStep == step) ? "active" : "")" href="javascript: void(0)"
```

Comment:
Empty href is worse than this?

Reply:
an empty line redirects us to the main page